test/unit/src/site/_filters/algolia-item.js

```diff
@@ -5,18 +5,35 @@ const {
   imgixDomain,
 } = require('../../../../../src/site/_data/site');
 const algoliaItem = require('../../../../../src/site/_filters/algolia-item');
+const {memoize} = require('../../../../../src/site/_filters/find-by-url');
 
 const createPost = () =>
   /** @type {EleventyCollectionItem} */ ({
     data: {
       lang: defaultLocale,
       title: 'Hello world',
+      page: {
+        filePathStem: '',
+      },
     },
     templateContent: 'Hello world',
     url: '/hello-world/',
   });
 
+/**
+ * Adds parent to memorized posts for following tests, this is required as get-post-parent-url
+ * is called by algoliaItem(post)
+ */
+const initParent = () => {
+  const mockParent = createPost();
+  mockParent.data.title = 'Parent Title';
+  mockParent.url = '/parent/';
+  memoize([mockParent]);
+};
+
```

Comment:
Can we revert this? 🙂 We shouldn't require `memoize` here.

Reply:
ah this was just because getPostParentURL now requires a parent to be memorised, but I'll just change the function code so it doesn't instead